### DISCUSSION TITLE
g-helper ambient mode issue

### DISCUSSION THREAD

**Discussion by mthoth** [seerge/g-helper#1736] [Q&A] [answered]
i just downloaded 0.137.0 on my Zephyrus G16 and the ambient mode doesn't seem to work properly. It only takes the average color from the very left side of the screen and nothing in the middle and right side. 
![20231213_145527](https://github.com/seerge/g-helper/assets/66431750/3fff3b0f-4a5b-44ce-aa4c-63352544b453)
![20231213_145551](https://github.com/seerge/g-helper/assets/66431750/d4ef58fb-c06c-4de8-95b4-dd885e626701)

**Comment by Kikimahe** [comment]
Hi,

On Tuf it's the same thing. (ver.134 too).

I guess that's how Ambient works to limit resources usage.

**Comment by seerge** [comment, marked as answer]
@mthoth @Kikimahe 

[GHelper.zip](https://github.com/seerge/g-helper/files/13662743/GHelper.zip)

Try this build

**Reply by Kikimahe** [reply]
OK on Tuf. Thanks Seerge this works so much better.

You are an incredibly responsive and talented guy.
If I wasn't already married to a woman I would start thinking about marrying you :D

Thanks again for everything you do :)

**Reply by mthoth** [reply]
works great on the zephyrus thanks so much :D
